src/main/java/com/cvsgo/repository/ReviewCustomRepository.java

```diff
@@ -1,19 +1,23 @@
 package com.cvsgo.repository;
 
+import com.cvsgo.dto.review.ReadProductReviewQueryDto;
+import com.cvsgo.dto.review.ReadProductReviewRequestDto;
 import com.cvsgo.dto.review.ReadReviewQueryDto;
 import com.cvsgo.dto.review.ReadReviewRequestDto;
-import com.cvsgo.dto.review.SearchReviewQueryDto;
-import com.cvsgo.dto.review.SearchReviewRequestDto;
 import com.cvsgo.entity.User;
 import java.util.List;
 import org.springframework.data.domain.Pageable;
 
 public interface ReviewCustomRepository {
-    List<SearchReviewQueryDto> searchByFilter(User user, SearchReviewRequestDto request, Pageable pageable);
 
-    List<ReadReviewQueryDto> findAllByProductIdAndFilter(User loginUser, Long productId,
-        ReadReviewRequestDto filter, Pageable pageable);
+    List<ReadReviewQueryDto> findAllByFilter(User user, ReadReviewRequestDto request,
```

Comment:
상품의 리뷰를 필터 적용해 조회하는 함수 `findAllByProductIdAndFilter`와 전체 리뷰를 필터 적용해 조회하는`findAllByFilter`와의 매개변수명을 통일하면 깔끔할 것 같아요!
```suggestion
    List<ReadReviewQueryDto> findAllByFilter(User user, ReadReviewRequestDto filter,
```